websites/M/MyOnlineRadio/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "MyOnlineRadio",
+	"description": {
+		"en": "online radio in over 30 countries"
+	},
+	"url": [
+		"myonlineradio.hu",
+		"myradioonline.ro",
+		"myonlineradio.sk",
+		"myonlineradio.de",
+		"ukradiolive.com",
+		"myonlineradio.at",
+		"myradioonline.it",
+		"myradioonline.es",
+		"myradioendirect.fr",
+		"irishradiolive.com",
+		"myradioonline.pl",
+		"myonlineradio.nl",
+		"myradioonline.cl",
+		"myradioenvivo.ar",
+		"myradioenvivo.mx"
+	],
+	"version": "1.0.0",
```

Comment:
There are new fields called `matches`, which are required for new Presences .
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.

Reply:
ohhhh so like `https://myonlineradio.hu/*`?

Reply:
yeah

Reply:
they don't replace the `url` section, but are a new section for future versions of PreMiD